packages/grafana-ui/src/components/Pagination/Pagination.tsx

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import { css } from 'emotion';
 import { stylesFactory } from '../../themes';
 import { Button, ButtonVariant } from '../Button';
+import { Icon } from '../Icon/Icon';
+
+const PAGE_LENGTH_TO_CONDENSE = 10;
```

Comment:
Could this become a property of the Pagination component, so that it could be easily adjusted if necessary? We can still default to 10 :)